rotkehlchen/tests/fixtures/accounting.py

```diff
@@ -360,10 +364,11 @@ def fixture_inquirer(
         mocked_current_prices_with_oracles,
         current_price_oracles_order,
         ignore_mocked_prices_for,
+        ethereum_manager,
```

Comment:
This change makes many tests that use inquirer fail, since ethereum manager uses the, the ethereum inquirer which uses the database fixture. And as such the database is already initialized with the test_user and it fails when trying to initialize a new user DB from rotkehlchen instance fixture.